javascript/webdriver/BUILD.bazel

```diff
@@ -66,3 +66,12 @@ closure_test_suite(
         "//javascript/webdriver/atoms/inject:deps",
     ],
 )
+
+genrule(
+    name="create-cdp-srcs",
+    outs=["devtools/types"],
+    cmd = "node $(location protocol-dts-generator.js) $@",
+    tools=[
+        "/javascript/webdriver/devtools/generator/protocol-dts-generator.js",
```

Comment:
The actual tool is `node` here. The script should be included in the `srcs`. I suspect you also want the `browser_protocol.json` and `js_protocol.json` in the sources too. The trick is to imagine that only the things listed as inputs to the rule are actually available.

Since the `protocol-dts-generator.js` file is local to this build file, you _may_ need to prefix it with a `:`, to indicate that it's a target.